test/MINLPTests/Project.toml

```diff
@@ -10,12 +10,12 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Uno_jll = "396d5378-14f1-5ab1-981d-48acd51740ed"
 
 [compat]
-Bonmin_jll = "100.800.901"
+Bonmin_jll = "100.800.902"
 Couenne_jll = "0.500.802"
 Ipopt_jll = "300.1400.1900"
 JuMP = "1"
 MathOptInterface = "1.33"
 MINLPTests = "0.6"
 SHOT_jll = "100.100.0"
-Uno_jll = "1"
+Uno_jll = "2"
```

Comment:
We fixed so many bugs before that I highly suggest `2.3.0`.

Reply:
Yes, I just want to be sure that the tests here are always using Uno >= 2.3.0 in the future.